pyproject.toml

```diff
@@ -0,0 +1,53 @@
+[build-system]
+requires = ["hatchling", "hatch-vcs"]
+build-backend = "hatchling.build"
+
+[project]
+name = "trollflow2"
+dynamic = ["version"]
+description = "Pytroll workflow execution framework"
+readme = "README.md"
+license = { text = "GPLv3+" }
+requires-python = ">=3.9"
+authors = [
+    { name = "Martin Raspaud", email = "martin.raspaud@smhi.se" },
+]
```

Comment:
Should there be more authors listed?

Reply:
Absolutely, I just did an automatic migration with hatch for the most, didn't look too much at the metadata sorry